APE18.rst

```diff
@@ -0,0 +1,53 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+APE 10 previously determined supported CPython versions, and provided a list up to the 4.0 release.
+Since APE 10 was written the wider scientific Python community have written and largely adopted `NEP 29`_ as a policy for supported versions of both CPython and Numpy.
+This APE proposes that Astropy adopt this policy following other projects in the scientific ecosystem.
+
+
+Detailed description
+--------------------
+
+This APE fully adopts `NEP 29`_ for both CPython and Numpy versions, it supersedes APE 10 wherever there is a conflict between that APE and this APE or `NEP 29`_.
+
+While this APE will not replicate the whole of `NEP 29`_, and `NEP 29`_ should be considered the canonical source for the support policy, the core scheme to be adopted is as follows:
+
+The core astropy package will support:
```

Comment:
One thing I'm confused by here: is this at the time of release or at the time of latest *major version* release?  That is, if we do a 4.0.x release 18 months after 4.0 is released, are we bound by the Python version from 42 months before 4.0.x or 60 months?

Reply:
There are some graphics in NEP29, that makes this clear.

Reply:
(except the ascii rendering is messed up, the text still makes it clear)

Reply:
It's based on the *scheduled* release date of non-bugfix versions, so a LTS version will support the version at the scheduled release for it's lifetime. So it would be 60 months in your example.

Reply:
Gotcha, that definitely makes sense, but I was suggesting that we state this explicitly.  I'll make a concrete suggest for that shortly in a follow-up comment.